### PR TITLE
also fix url

### PR DESCRIPTION
**Pull request by patrickkettner**
to fully fix for #5289

last commit wasn't included in the first PR (#6245)

### CODE REVIEW

**Comment by patrickkettner** [comment]
@sebastianbenz thanks so much for the quick merge, but I forgot this bit

File: examples/source/interactivity-dynamic-content/Subscription_Settings/index.html [hunk 1]
```diff
@@ -112,17 +112,20 @@
           {
             "value": "watching",
             "isSelected": false,
-            "text": "Watching"
+            "text": "Watching",
+            "imgUrl": "/images/watching.jpg"
```
Comment (sebastianbenz) [inline examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:116]: Do this images exist?

Comment (patrickkettner) [reply examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:116]: The images that are actually used in the code to exist, the ones in the comments just shortened fake ones for the sake of brevity/readability. If you think the value having them be the actual your URLs is higher I can change it

Comment (sebastianbenz) [reply examples/source/interactivity-dynamic-content/Subscription_Settings/index.html:116]: got it

**Review by sebastianbenz — APPROVED** [review]

**Pull request merged**